### PR TITLE
Removed HTTP Plugin Docs & Reference (SDK Docs)- Closes 1542

### LINKED ISSUE #1542: Remove Sequence & Keys module and the HTTP API plugin
### Description

- Remove sequence & keys modules docs
- Remove HTTP API plugin docs

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1542

### CODE REVIEW

File: modules/ROOT/pages/plugins/http-api-plugin.adoc [hunk 1]
```diff
@@ -1,1341 +1,1341 @@
-= HTTP API plugin
```
Comment (Tschakki) [inline modules/ROOT/pages/plugins/http-api-plugin.adoc:1]: Please remove the complete page directly

**Review by Tschakki — CHANGES_REQUESTED** [review]
Just one small comment, rest looks good to me!

File: modules/ROOT/pages/plugins/http-api-plugin.adoc (hunk 1 above)
Comment (TalhaMaliktz) [reply modules/ROOT/pages/plugins/http-api-plugin.adoc:1]: Alright done

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**